cmd/lotus-storage-miner/run.go

```diff
@@ -60,6 +58,10 @@ var runCmd = &cli.Command{
 			return err
 		}
 
+		if _, _, err := ulimit.ManageFdLimit(); err != nil {
```

Comment:
Added a flag allowing you to disable this (like in go-ipfs)

I've added it here because the default is usually too low to be useful, I'm yet to see a good reason to keep this limit low